packages/compass-indexes/src/components/create-index-form/index-flow-section.spec.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { render, screen } from '@mongodb-js/testing-library-compass';
+import IndexFlowSection from './index-flow-section';
+import { expect } from 'chai';
+
+describe('IndexFlowSection', () => {
+  const renderComponent = (createIndexFieldsComponent?: JSX.Element) => {
+    render(
+      <IndexFlowSection
+        createIndexFieldsComponent={createIndexFieldsComponent ?? null}
+      />
+    );
+  };
+  it('renders the Input Index header', () => {
+    renderComponent();
+    expect(screen.getByText('Input Index')).to.exist;
```

Comment:
Totally fine as is, but I figured I'd mention, instead of `.to.exist` I'll typically use `.to.be.visible`. That way it's sure it's not accidentally hidden to the user but exists.
https://www.chaijs.com/plugins/chai-dom/